src/__tests__/__helpers__/renderWithProviders.tsx

```diff
@@ -77,7 +77,7 @@ const providersByName: {
   overlay: [OverlayProvider],
   toast: [ToastProvider],
   menu: [MenuProvider],
-  user: [UserProvider, { user: undefined }],
+  user: [UserProvider, { user: undefined, fetcher: async () => undefined }],
```

Comment:
I realised that some tests were making a request to load the user, this stops that.